src/front/cobalt.js

```diff
@@ -703,6 +705,6 @@ document.onkeydown = (e) => {
         if (e.key === "M") popup('settings', 1);
         
     } else {
-        if (e.key === "Escape") hideAllPopups();
+        if (e.key === "Escape" || e.key === "Enter") hideAllPopups();
```

Comment:
this will close all popups no matter the context, such as from settings/about screen. i recommend changing the implementation to be applied only to error message!